client/signup/steps/site-or-domain/index.jsx

```diff
@@ -47,9 +51,25 @@ export default class SiteOrDomain extends Component {
 	handleClickChoice( event, designType ) {
 		event.preventDefault();
 
-		const { stepName, goToStep, goToNextStep } = this.props;
+		const {
+			stepName,
+			goToStep,
+			goToNextStep,
+			queryObject,
+			translate,
+		} = this.props;
 
-		SignupActions.submitSignupStep( { stepName }, [], { designType } );
+
+		if ( ! queryObject || ! queryObject.domain ) {
+			notices.error( translate( 'Invalid domain name.' ) );
```

Comment:
Hi! I've found a possible matching string that has already been translated **15** times:
`translate( 'Invalid Domain Name' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).